### DISCUSSION TITLE
How do I get WiFi working?

### DISCUSSION THREAD

**Discussion by hexisXz** [helloSystem/hello#488] [Q&A] [answered]
So I am a long time Linux user and I got bored so I looked at distro watch and found this project. I installed it on my laptop and HOlY I have never seen something this good. so since I installed it on my laptop I need wifi but I have absolutely no idea how to set it up in the wifi utility so how would I do so?

**Comment by hexisXz** [comment, marked as answer]
Nevermind I figured it out

**Reply by hardRockWorld** [reply]
How did you do that?
